channels_redis/core.py

```diff
@@ -384,7 +384,11 @@ def _backup_channel_name(self, channel):
                         # Ensure all tasks are cancelled if we are cancelled.
                         # Also see: https://bugs.python.org/issue23859
                         for task in tasks:
-                            task.cancel()
+                            cancelOK = task.cancel()
+                            if not cancelOK:
```

Comment:
You can combine `task.cancel` into the `if not` statement here and avoid the variable

Reply:
Thanks Andrew, cleanups committed.